src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
What does htis do?

Reply:
This is default React functionality that can be used to measure the performance and user experience of the application. Maybe useful during development, will probably never be active when deployed: https://create-react-app.dev/docs/measuring-performance/

Reply:
Ok sounds good!